### DISCUSSION TITLE
Help With Deploying Dapp

### DISCUSSION THREAD

**Discussion by ericrainford** [hashlips-lab/nft-erc721-collection#308] [Q&A]
Hey everyone I hope someone can help me out here. for starters I don't know how to code and just wanted to make my own NFT collection for fun, I've followed hashlips masterclass and have gotten everything to work on a local testnet, at the end of the video hashlips says to copy the build folder and index.html to the root directory of your website or server, this sounds like its common sense to everyone in here but I'm lost and want to know where to start

I don't have a website and what I have tried doing is putting files into a github repo under my account then using services like netlify & vercel but all I can get are 404's or a white page with "logo" in the top corner

I feel like I can't give up now as I've spent so much time on this project so any help is appreciated looking for what I am doing wrong on netlify & vercel or how to deploy my dapp?/make a live mint site

**Comment by EverDegen** [comment]
I can 100% relate to this it's like my case. So I followed the last video of this series (Dapp deployment) and got my minting dapp deployed successfully on cloudflare.
On the other hand after I deployed it to cloudflare I thought why not try out netlify also so I deployed it there but it didn't work.
So I recommend you to go through the video again and replicate it. If your dapp working on local machine it will also work on cloudflare.

I have never used Git and GitHub before, this was my first attempt and luckily a successful one.
here is the result: https://fantomkites.com

liarco is the best teacher tbh he's always on point, no time waste. A 10 min video tutorial of liarco = 30 min of other's tutorial.

Also I want to add one last thing that I used git to push my codes to GitHub. So if you're uploading files manually to GitHub I would suggest you to use git. You can find tutorials on how to use it.
After that just follow the video (dapp deployment by liarco) and you will get success hopefully.
